graphium/cli/train_finetune_test.py

```diff
@@ -266,10 +267,11 @@ def run_training_finetuning_testing(cfg: DictConfig) -> None:
     logger.info("Total compute time:", timeit.default_timer() - st)
     logger.info("-" * 50)
 
-    if wandb_cfg is not None:
+    save_checkpoint_to_wandb = cfg["trainer"].get("save_checkpoint_to_wandb")
+    if save_checkpoint_to_wandb is True:
         # Save initial model state - and upload checkpoint to wandb
         if cfg["trainer"]["model_checkpoint"]["save_last"] is True:
-            checkpoint_path = f"{cfg['trainer']['model_checkpoint']['dirpath']}{cfg['trainer']['model_checkpoint']['filename']}.ckpt"
+            checkpoint_path = f"{cfg['trainer']['model_checkpoint']['dirpath']}/{cfg['trainer']['model_checkpoint']['filename']}-v1.ckpt"
```

Comment:
Maybe a glob would be better, so we capture all checkpoints?

It's probably worth noting that the `filename` parameter for a W&B parameter can actually be a template, so this might not work if a user does that (ref: https://lightning.ai/docs/pytorch/stable/api/lightning.pytorch.callbacks.ModelCheckpoint.html)

Reply:
So because I added the timestamp to the directory name, there will never be multiple checkpoints in the same folder, so not sure if glob is needed. We could use `last.ckpt` as it seems to be identical to the filename one